config/config.h.tmp

```diff
@@ -20,10 +20,11 @@
 
 #include <IBAMR_config.h>
 
+// Ultimately, all IBAMR macros will be aliases of the IBTK macros
```

Comment:
Are you planning to keep IBTK as a separate package? Should we just roll everything into IBAMR and provide backward compatibility by making IBTK macros aliases to IBAMR macros?

Reply:
They are no longer separate projects (as far as the build system is concerned) in the CMake build system, so no (though the directory structure is unchanged). Since IBAMR depends on IBTK I set up the macros for IBTK first.

We can do it either way but it doesn't really impact this patch - that's a change that will be made in the cmake code.

Reply:
The original idea was that IBTK provided lower level discretization tools that could be used to put together different kinds of "immersed" models, with IBAMR being the implementation of a "standard" IB method. For instance, at one point, I actively used an electrophysiology simulation code that used IBTK but not IBAMR.

I've tried to maintain this separation, in which more generic things wind up in IBTK, but I am not sure whether it is really a useful distinction.

Reply:
We might want to continue this discussion in #1020 - this isn't the way its set up right now but it would not be hard to split them off since the CMake build system defines `IBTK2d` and `IBTK3d` as distinct targets with their own properties, so in principle they could be split off and installed separately pretty easily (though that is currently not supported).